packages/react-core/src/components/Tabs/Tabs.tsx

```diff
@@ -133,6 +176,19 @@ export class Tabs extends React.Component<TabsProps & OUIAProps, TabsState> {
     }
   };
 
+  // Format the inset prop string by capitalizing the first letter.  If the string prop is '2xl' append '_' to beginning of the string so we can key the correct modifier.
+  formatInsetString = (s: string) => (s === '2xl' ? '_2xl' : capitalize(s));
+
+  componentDidMount() {
+    window.addEventListener('resize', this.handleScrollButtons, false);
+    // call the handle resize function to check if scroll buttons should be shown
+    this.handleScrollButtons();
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('resize', this.handleScrollButtons, false);
```

Comment:
in componentDidMount you added the listener to window. They both should be the same

Reply:
Interesting this is how it was before.  I did not update here.  Should we have it on window on both?

Reply:
Both should be on `window`.